src/environment.rs

```diff
@@ -12,7 +12,10 @@ pub enum ResultCode {
 #[derive(Clone, Debug, PartialEq)]
 pub enum Environment {
     System(SystemEnvironment),
-    Rcc(RCCEnvironment),
+    Rcc(
+        // see https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant
```

Comment:
It says always measure the changes this lint suggests ? From what I can tell boxing is a premature optimization here, I would just suppress it.

Reply:
Done